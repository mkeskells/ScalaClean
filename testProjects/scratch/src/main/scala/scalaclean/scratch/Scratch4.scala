object X extends App {
  val (a,b,c) = (1,2,3)

  val (d:Int, e:Int, f, g:Int) = (1,2,3, 4)

  println(e)
}