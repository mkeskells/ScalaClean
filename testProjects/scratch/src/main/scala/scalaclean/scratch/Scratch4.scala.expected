object X extends App {

  /* SCALA CLEAN consider rewriting pattern as 3 values are not used */val (_:Int, e:Int, _, _:Int) = (1,2,3, 4)

  println(e)
}